internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -96,7 +96,7 @@ public class VeniceWriter<K, V, U> extends AbstractVeniceWriter<K, V, U> {
   private final ThreadPoolExecutor threadPoolExecutor;
 
   // log4j logger
-  private final Logger logger;
+  protected final Logger logger;
```

Comment:
I'm not finding it, why open this up?

Reply:
In `ComplexVeniceWriter`'s `putLargeValue`
```
  @Override
  protected CompletableFuture<PubSubProduceResult> putLargeValue(
      byte[] serializedKey,
      byte[] serializedValue,
      int valueSchemaId,
      PubSubProducerCallback callback,
      int partition,
      LeaderMetadataWrapper leaderMetadataWrapper,
      long logicalTs,
      PutMetadata putMetadata,
      ChunkedValueManifest oldValueManifest,
      ChunkedValueManifest oldRmdManifest) {
    skippedLargeRecords.incrementAndGet();
    if (!REDUNDANT_LOGGING_FILTER.isRedundantException(topicName, SKIP_LARGE_RECORD)) {
      logger.warn("Skipped writing {} large record(s) to topic: {}", skippedLargeRecords.getAndSet(0), topicName);
    }
    return CompletableFuture.completedFuture(null);
  }
```